modules/carto/src/api/maps-v3-client.js

```diff
@@ -69,8 +70,8 @@ function dealWithError({response, error}) {
 /**
  * Build a URL with all required parameters
  */
-function getParameters({type, source, geoColumn, columns, schema}) {
-  const parameters = [encodeParameter('client', 'deck-gl-carto')];
+function getParameters({type, source, geoColumn, columns, schema, client}) {
```

Comment:
I prefer to call it clientId rather than client in all the places. client could be confusing because it can be understood as a class